src/main/java/lotto/LottoNumbers.java

```diff
@@ -0,0 +1,63 @@
+package lotto;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class LottoNumbers {
+    public static final int MAX_SIZE = 6;
+    public static final int LOTTO_LIMIT_NUMBER = 46;
+    private List<Integer> lottoNumbers;
+    private LottoPrice lottoPrice;
+
+    public LottoNumbers() {
+        this.lottoNumbers = new ArrayList<>();
+        for (int i = 0; i < LOTTO_LIMIT_NUMBER; i++) {
+            lottoNumbers.add(i);
+        }
+        this.lottoPrice = new LottoPrice();
+    }
+
+    public LottoNumbers(List<Integer> lottoNumbers) {
+        this.lottoNumbers = lottoNumbers;
+    }
+
+    /**
+     * 로또번호를 제공
+     * @return
+     */
+    public void mark() {
+        Collections.shuffle(lottoNumbers);
+        sixNumbers(lottoNumbers);
+    }
+
+    /**
+     * 6개의 번호를 전달
+     * @param shuffledNumbers
+     * @return
+     */
+    private void sixNumbers(List<Integer> shuffledNumbers) {
```

Comment:
```suggestion
    private void initializeLottoNumbers(List<Integer> shuffledNumbers) {
```

이런 메서드 명은 어떠신가요?
메서드를 일반 명사로 지으면, 맴버인지 메서드인지 한눈에 파악하기 쉽지 않습니다.

Reply:
반면 이 부분은 제 피드백을 유지해야할 것 같습니다.
void 메서드는 manipulator 로 분류되어야 하니깐요.

다만 initialize 라는 이름은, 프로시져 지향적인 안좋은 동사 같습니다.
저보다 더 좋은 동사를 떠올려서 다시 명명해주세요!